pkgs/development/python-modules/tinydb/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pytestcov
+, pytestrunner
+, pycodestyle
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "tinydb";
+  version = "v3.14.1";
+
+  src = fetchFromGitHub {
+    owner = "msiemens";
+    repo = pname;
+    rev = version;
+    sha256 = "02idbvrm8j4mwsjfkzy11f4png19k307p53s4qa2ifzssysxpb96";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    pytestrunner
```

pytestrunner should go in nativeBuildInputs
```suggestion
    pytestrunner
```

---

Thanks, I also added `doCheck = true;`, I didn't realise that was false by default.

---

for buildPythonPackage, the default is `doCheck= true;` , for mkDerivation it's defaulted to false

---

Damn, saw that just after I pushed the latest commit. I'll just leave `doCheck` in for now, I'll clean it up if this PR needs another commit.